### PR TITLE
Constrain Oml 0.0.6 and 0.0.7 dependencies

### PR DESCRIPTION
**Pull request by rleonid**
The impetus for this PR was to constrain the version of `lacaml` due to recent changes. I also took the opportunity to constrain the dependencies in 0.0.7 that were left off on the other packages (lbfgs, ocephes).

### CODE REVIEW

**Comment by hannesm** [comment]
you can negate the version constraint and put it into `conflicts`.

**Comment by dbuenzli** [comment]
@releonid IIRC the fact that depopts no longer support version is explained in this document https://github.com/ocaml/opam/blob/master/doc/design/depopts-and-features

**Comment by rleonid** [comment]
Thanks for the pointer on the documentation and the easy solution. Took a couple of tries to figure out the correct way to express the negation (no formula's in conflicts!). I apologize if my force pushing messed up the CI.

File: packages/oml/oml.0.0.6/opam [hunk 1]
```diff
@@ -12,7 +12,13 @@ remove: ["ocamlfind" "remove" "oml"]
...
   "cppo" { build & >= "1.4.0" }
-  "lacaml" { >= "8.0.6" }
-  "lbfgs" { >= "0.8.7" }
-  "ocephes" { >= "0.8" }
+  "lacaml"
+  "lbfgs"
+  "ocephes"
+]
+conflicts : [
+  "lacaml" { <= "8.0.6" }
```
Comment (avsm) [inline packages/oml/oml.0.0.6/opam]: This isn't quite equivalent to the previous dependency -- did you mean `<"8.0.6"`

Comment (rleonid) [inline packages/oml/oml.0.0.6/opam]: Yes, you're correct!

**Pull request merged**